resources/lang/pl_PL/models.yml

```diff
@@ -0,0 +1,57 @@
+generic:
+  createdAt: "utworzono"
+  modifiedAt: "zmodyfikowano"
+  name: "nazwa"
+  email: "adres e-mail"
+  createdBy: "utworzono przez"
+dictionary:
+  _name: "słownik"
+  _names: "słowniki"
+  name: "$t(models.generic.name)"
+  facility: "placówka, właściciel słownika"
+  isFixed: "nieusuwalny / systemowy"
+  positions: "pozycje"
+position:
+  _name: "pozycja słownika"
+  _names: "pozycje słownika"
+  name: "$t(models.generic.name)"
+  dictionary: "słownik"
+  facility: "placówka, właściciel pozycji"
+  isFixed: "nieusuwalna / systemowa"
+  defaultOrder: "domyślna kolejność"
+facility:
+  _name: "placówka"
+  _names: "placówki"
+  name: "$t(models.generic.name)"
+  url: "adres url wewnątrz Memo"
+  members: "osoby przypisane do placówki"
+member:
+  _name: "powiązanie osoby i placówki"
+  _names: "powiązania osób i placówek"
+  user: "użytkownik"
+  facility: "placówka"
+  hasFacilityAdmin: "administrator placówki"
+permissions:
+  _name: "uprawienia"
```

Comment:
A tu to nagle nie ma plurala? :)

Reply:
bo to występuje tylko w liczbie mnogiej, musicie sobie dać radę tak